tasks/broad/JointGenotypingTasks.wdl

```diff
@@ -1077,4 +1077,41 @@ task PartitionSampleNameMap {
     disks: "local-disk 10 HDD"
     docker: "us.gcr.io/broad-gotc-prod/python:2.7"
   }
-}
\ No newline at end of file
+}
+
+# Task used to divide annotations that were summed by GenomicsDB to get average values.
+task CalculateAverageAnnotations {
+  input {
+    File vcf_index
+    File vcf
+    Array[String] annotations_to_divide = ["ASSEMBLED_HAPS", "FILTERED_HAPS", "TREE_SCORE"]
+
+    String docker = "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:UG_feature_branch_v4"
```

Comment:
It is as of this week! However, this is the docker we've been using in all of the Ultima pipelines. It seems like it might be better to update them all at once in a separate PR? Happy to hear if you think I should update it here first.